bot/extensions/custom_roles/events.py

```diff
@@ -16,9 +15,11 @@ def __init__(self, bot: core.DiscordBot):
         self.bot = bot
         self.updated_at: dict[int, datetime] = {}
 
-    @property
-    def custom_roles_logs_channel(self) -> discord.TextChannel | None:
-        return self.bot.guild.get_channel(settings.custom_roles.log_channel_id)
+    async def get_custom_roles_logs_channel(self):
+        query = """SELECT * FROM configs
+                    WHERE guild_id = $1"""
+        data = await Config.fetchrow(query, self.bot.guild.id)
+        return self.bot.guild.get_channel(data.custom_role_log_channel_id)
```

Comment:
Can't fetch channel just from bot.guild as the channel can be in a different guild.

This function should take in a guild_id parameter and return the channel for that.
Also, instead of selecting everything you can select only the `custom_role_log_channel_id` column

Reply:
yes you can, use fetchval here.